src/common.h

```diff
@@ -82,7 +82,7 @@ static inline int msg_sendto(int fd, uint8_t *buf, int nbytes,
  * the MSG_NOSIGNAL flag.
  * Note: another way is call signal (SIGPIPE,SIG_IGN);
  */
-#ifndef WIN32
+#ifndef __windows__
```

Comment:
Please mark also a corresponding `#endif`.

Reply:
not sure what do you mean 
![ifdef](https://user-images.githubusercontent.com/44922030/183436355-3254b504-075d-4446-9afc-b0cfa73d2451.JPG)

Reply:
```
#endif // __windows__
```